.github/workflows/next-build-nightly.yml

```
@@ -0,0 +1,155 @@
+name: next-build-nightly
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [20, 22]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: 'main'
+      - run: git fetch --depth=1
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+      - run: yarn install --frozen-lockfile --ignore-engines
+      - run: yarn test:unit
+        env:
+          SANDBOX_TOKEN: ${{ secrets.API_HUB_SANDBOX_TOKEN }}
+      - run: yarn test:integration
+      - run: yarn test:self
+      - run: yarn test:build-packages
+      - run: yarn test:type
+      - if: failure()
+        name: Slack Notify
+        uses: rtCamp/action-slack-notify@v2.3.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          SLACK_USERNAME: SDK Pipeline Bot
+          SLACK_TITLE: Build tests for v4 ${{ job.status }}
```

```suggestion
          SLACK_TITLE: Build tests for v4 ${{ job.status }} (${{ matrix.version }})
```
[req] missing ` (${{ matrix.version }})`?

---

Did you mean `matrix.node-version` ? I have added that, other than that there is no other applicable `version` property here.